frontend/src/components/ProjectSelectionComponent.vue

```diff
@@ -17,50 +15,82 @@
 import Vue from "vue";
 import apiService from "@/services/api.service";
 import Project from "../model/Project";
+import autocomplete from "vue2-autocomplete-input-tag";
 
 export default Vue.extend({
   name: "ProjectSelectionComponent",
 
+  components: {
+    autocomplete,
+  },
+
   data() {
     return {
       selected: null,
-      project: "",
+      projectArray: [] as Array<Project>,
+      input: "",
+      aCorrectProject: false,
     };
   },
   computed: {
     projects(): Array<Project> {
       return this.$store.state.projects;
     },
-    getAllProjects(): Array<unknown> {
-      const projectNames = this.projects.map((p) => p.name);
-      if (projectNames.length < 1) {
-        return [
-          {
-            value: null,
-            text: this.$t("session.prepare.step.selection.mode.description.withJira.formSelection"),
-          },
-        ];
-      } else {
-        return [
-          {
-            value: null,
-            text: this.$t("session.prepare.step.selection.mode.description.withJira.formSelection"),
-          },
-          ...projectNames,
-        ];
-      }
+
+    getProjectNames(): Array<string> {
+      let projectNames;
+      projectNames = this.projects.map((p) => p.name);
+      return projectNames;
     },
   },
 
   methods: {
-    async getUserStories(project) {
-      console.log(`Trying to select ${project}`);
-      const selectedProject = this.projects.find((p) => p.name === project);
-      console.log(`Selected: ${selectedProject}`);
-      this.$store.commit("setSelectedProject", selectedProject);
-      const response = await apiService.getUserStoriesFromProject(project);
-      this.$store.commit("setUserStories", { stories: response });
+    async getUserStories() {
+      this.aCorrectProject = false;
+      console.log(`Trying to select ${this.selected}`);
```

Comment:
This seems to be spamming in the console because it gets triggered when the input changes

Reply:
Remove console.log for productive code.

Reply:
Hi @stritti,

console.log is disabled for production -> [Source](https://github.com/Sybit-Education/Diveni/blob/main/frontend/babel.config.js#L2-L3)